Tools/clinic/clinic.py

```diff
@@ -5281,16 +5292,8 @@ def state_parameter_docstring(self, line: str) -> None:
             return self.next(self.state_function_docstring, line)
 
         assert self.function and self.function.parameters
-        last_parameter = next(reversed(list(self.function.parameters.values())))
-
-        new_docstring = last_parameter.docstring
-
-        if new_docstring:
-            new_docstring += '\n'
-        if stripped:
-            new_docstring += self.indent.dedent(line)
-
-        last_parameter.docstring = new_docstring
+        last_param = list(self.function.parameters.values())[-1]
```

Comment:
I think it's more idiomatic to do as the existing code does, and call `next(reversed(it))` to get the last value of an iterable. But we can drop the unnecessary cast to `list()` and iterate directly over the values:

```suggestion
        last_param = next(reversed(self.function.parameters.values()))
```

Reply:
Yeah, `next(reversed(it))` is fine